src/compiler/crystal/command.cr

```diff
@@ -446,6 +446,16 @@ class Crystal::Command
         end
       end
 
+      unless no_codegen
+        opts.on("--frame-pointers auto|always|non-leaf", "Control the preservation of frame pointers") do |frame_pointers_s|
+          if frame_pointers = FramePointers.parse? frame_pointers_s
+            compiler.frame_pointers = frame_pointers
+          else
+            error "Invalid value `#{frame_pointers_s}` for frame-pointers"
+          end
+        end
+      end
```

Comment:
**question:** Why place this in a separate position? I think it would fit well within a `unless no_codegen` block above. Maybe after the debug options would be a good space that keeps related options together?

Reply:
I...think I was trying to keep it in rough alphabetic order...but also put `f` before `e` anyway? idk, but I did just place it with the debug options now.